types/react-input-mask-format/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for react-input-mask-format 1.0
+// Project: https://github.com/temirtator/react-input-mask-format
+// Definitions by: Alexandre Paré <https://github.com/apare>
```

Comment:
Do these people know they're being signed up as maintainers?  Should you be in the list?